test/views/ViewSpec.scala

```diff
@@ -33,14 +33,16 @@ class ViewSpec extends WordSpec with MustMatchers with ViewTemplates with ViewMa
 
   implicit protected def htmlBodyOf(html: Html): Document = Jsoup.parse(html.toString())
 
-  private val messagesApi: MessagesApi = instanceOf[MessagesApi]
+  private val messagesApi: MessagesApi = ViewSpec.realMessagesApi
 
   protected implicit def messages(implicit request: Request[_]): Messages =
     new AllMessageKeysAreMandatoryMessages(messagesApi.preferred(request))
 
-  protected implicit def messages(key: String)(implicit request: Request[_]): String = messages(request)(key)
+  protected def messages(key: String, args: Any*)(implicit request: Request[_]): String = messages(request)(key, args: _*)
```

Comment:
Why has this changed?

Reply:
It is not required to be implicit. The messages above do, as they are used in twirl templates.
But these here are only ever used explicitly.